src/Microsoft.Azure.ServiceBus/Management/MessagingSku.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Management
+{
+    /// <summary>
+    /// Specifies the SKU/tier of the messaging namespace.
+    /// </summary>
+    public enum MessagingSku
```

Comment:
`MessagingSku` --> `NamespaceSku`
Messaging SKU is vague. These are specific to ASB namespaces.

Reply:
By popular vote internally, I'm keeping this as-is. There is some reuse of code that we do and this sku enum is used across services. 

---
In reply to: [224154533](https://github.com/Azure/azure-service-bus-dotnet/pull/585#discussion_r224154533) [](ancestors = 224154533)